docs/philosophy.md

```diff
@@ -20,6 +20,16 @@ I have a [video](https://www.youtube.com/watch?v=N-eTcjGsK08) where I break down
 - The best way to maximize my own experience is to help maximize others' experience as well as my own. I believe that humans have a unique capability for reciprocity, and that given this universal goal, pooling the efforts of a community of people will exponentially increase progress towards it. I (and most other people) also lose quality of life when those around me do (e.g. seeing people homeless, having friends with poor mental health, having ill family members, etc).
 - Therefore, the best rules in a society are rules that provide the best outcomes for the greatest number of people. In philosophical jargon terms, this would make me a [Rule Consequestialist](https://plato.stanford.edu/entries/consequentialism-rule/).
 
+## Moral Anti-realism
+
+The first foundational belief listed above is that "There is no such thing as morality", for which the philosophical term is [Moral Anti-realism](https://plato.stanford.edu/entries/moral-anti-realism/). In general, I am not a fan of conversations about [meta-ethics](https://plato.stanford.edu/entries/metaethics/) because I believe they generally reduce to incommensurable systems that become impossible to have a debate about. Instead, it seems that people are won over by whether such systems are intuitively satisfactory to them. I recognise, however, that to most people moral anti-realism seems like a dubious system as it is common to believe in some sort of moral truth, so it warrants a little more explanation here.
```

Comment:
```suggestion
The first foundational belief listed above is that "There is no such thing as morality", for which the philosophical term is [Moral Anti-realism](https://plato.stanford.edu/entries/moral-anti-realism/). In general, I am not a fan of conversations about [meta-ethics](https://plato.stanford.edu/entries/metaethics/) because I believe they generally reduce to incommensurable systems that become impossible to have a debate about. Instead, it seems that people are won over by whether such systems are intuitively satisfactory to them. I recognize, however, that to most people moral anti-realism seems like a dubious system as it is common to believe in some sort of moral truth, so it warrants a little more explanation here.
```